Gandalan.IDAS.WebApi.Data/DTOs/Produktion/EtikettDTO.cs

```diff
@@ -11,6 +11,7 @@ public class EtikettDTO
         public Guid EtikettGuid { get; set; }
         public string Kuerzel { get; set; }
         public string Text { get; set; }
+        public Guid Kennzeichen { get; set; }
```

Comment:
Please remove, will be removed from I1 as well (Ticket Devops#6077)

Reply:
```suggestion
```